style.css

```diff
@@ -0,0 +1,17 @@
+.container {
+	max-width: 1024px;
+	margin: 0 auto;
+}
+img {
```

Comment:
The closing brace is written on a new line and without an indent. The next rule after this is separated by an empty line. See more in this guide: http://codeguide.academy/html-css.html#html-language